Changelog.md

```diff
@@ -3,12 +3,14 @@
 ## Added
 
 -   Port selection for devices with multiple serialports.
+-   Selected ports are remembered per device.
```

Comment:
```suggestion
-   Selected ports are persisted per device.
```

Reply:
I'm fairly certain we had a similar discussion previously in the team and that "persisted" was not something the customer relates to well.
I'll double-check with Bjørn Inge